components/cardBuilder.vue

```diff
@@ -0,0 +1,36 @@
+<template>
```

Comment:
This component should be named `Card` instead, since it's a simple card :)

No need to keep the legacy of jf-web over here :yum:

Reply:
lol, I just copied the name :p